alembic/versions/da2582d3d2eb_initial.py

```diff
@@ -0,0 +1,198 @@
+"""Initial
+
+Revision ID: da2582d3d2eb
+Revises: 
+Create Date: 2021-10-16 15:54:56.336879
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'da2582d3d2eb'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
I had rather have a separate migration for each of the table, so that it is easier to change them when needed. Secondly you need to fix the indentations on them.

Reply:
I'm not sure you understand. If you want to change something on the database models you create a new migration, instead of modifying the current ones.
That's just an initial migration.

Reply:
Yes, this is the correct way to do things: an initial migration with all the data in it. If we make adjustments to the live database later, we need to:

- Add a new migration to the API project
- Mark the live database (currently used by the Django API) as migrated already.

That said, I believe the migration here has the wrong schema since we want to work with the existing Django data, and the current table / index names do not correspond to the correct values in Django, as Django namespaces table names with the Django application name prefix (such as `api_infraction`).